package/native-package/src/index.js

```diff
@@ -247,6 +249,26 @@ registerNativeHandlers({
       ignoreAndroidSystemSettings: false,
     });
   },
+  // eslint-disable-next-line react/display-name
+  Video: ({ onBuffer, onEnd, onLoad, onProgress, paused, resizeMode, style, uri, videoRef }) => (
+    <Video
+      onBuffer={onBuffer}
+      onEnd={onEnd}
+      onError={(error) => {
+        console.log(error);
+      }}
+      onLoad={onLoad}
+      onProgress={onProgress}
+      paused={paused}
+      poster='https://wallpaperaccess.com/full/1754609.jpg'
```

Comment:
We should have some photo we are hosting ourselves if we want this to be a static image. If the URL suddenly points to something else, or wallpaperaccess.com was to go down some time or whatever, we wouldn't be in control of it at all

Reply:
Dropping the poster prop for now as the URL with the image in stream CDN doesn't work at all when it is passed to the `poster` prop.

Reply:
Yes, lets handle it separately.